testplan/testing/multitest/entries/stdout/base.py

```diff
@@ -125,6 +125,12 @@ def get_details(self, entry):
 @registry.bind(base.TableLog)
 class TableLogRenderer(BaseRenderer):
     def get_details(self, entry):
```

Comment:
Please add the type annotations to the function touched. This is both a silent agreement within the team and it was clearly something we had to check by diving in.